src/utils/connection.ts

```diff
@@ -11,8 +11,12 @@ export function handleWsDashboard(storage: IStorage) {
   return (data: any) => {
     // update network height
     const height = data.best_block_height as number;
-    storage.setCurrentHeight(height);
-    storage.unlockUtxos(height);
+    storage.getCurrentHeight().then(async (currentHeight) => {
+      await storage.setCurrentHeight(height);
+      if (height !== currentHeight) {
+        await storage.unlockUtxos(height);
```

Comment:
Do we need to await?

Reply:
Not really, the initial idea was to run `unlockUtxos` before changing the height, but passing the height as a parameter is better, but it made the `await`s unnecessary, so I'm removing them